packages/react/src/FilteredActionList/useAnnouncements.tsx

```diff
@@ -36,17 +38,26 @@ const getItemWithActiveDescendant = (
 
   return {index, text, selected}
 }
+//TODO remove this when we remove usingRemoveActiveDescendant
 
 export const useAnnouncements = (
   items: FilteredActionListProps['items'],
   listContainerRef: React.RefObject<HTMLUListElement>,
   inputRef: React.RefObject<HTMLInputElement>,
   enabled: boolean = true,
   loading: boolean = false,
-  message?: {title: string; description: string},
+  message?: {title?: string; description?: string},
```

Comment:
can you explain why is this needed if the message is optional?

Reply:
Yeah I'm wondering why the title and description have to be optional now if they weren't before 🤔 , did that break anything?